assets/css/queries.css

```diff
@@ -52,6 +58,9 @@
   .hero {
     gap: 4.8rem;
   }
+  .btn-mobile-nav {
+    display: none !important;
```

Comment:
We should be careful with using `!important` in CSS files because it is very difficult to overwrite and can cause problems if the project grows in size. We should only use !important if there is no other way of doing things in CSS.

Reply:
Yes, I acknowledge. I am cautious now. Thanks